docs/README.md

```diff
@@ -0,0 +1,15 @@
+# DeepSpeed Documentation
+
+This directory includes the documentation of DeepSpeed. There are three ways to read it:
+
+## 1. Access [deepspeed.ai](https://www.deepspeed.ai/)
+
+This is the most recommended way to read the documentation.
+
+## 2. Directly read files in this directory
+
+We do not recommend this way because this directory is organized to build the [deepspeed.ai](https://www.deepspeed.ai/) website using Jekyll. Thus some of the files actually are not DeepSpeed documentation. In addition, some of the url links in the documentation only work through the webpages generated by Jekyll.
+
+## 3. Build [deepspeed.ai](https://www.deepspeed.ai/) website locally using Jekyll
+
+This is recommended when you do not have internet access. You can follow the instruction at [here](https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll) to install Ruby, Bundler, and Jekyll. Then run `bundle exec jekyll serve` at this directory so that you can view the website in you web browser at `http://localhost:4000`.
```

Comment:
Maybe also also add a mention to point 3 to say it's recommended for local website dev?

Reply:
Added in the new commit.